moment/moment-node.d.ts

```diff
@@ -115,6 +115,18 @@ declare module moment {
         toJSON(): string;
     }
 
+    interface MomentLocale {
+        ordinal(n: number): string;
+    }
+
+    interface MomentCreationData {
+        input?: string,
+        format?: string,
+        locale: MomentLocale,
+        isUTC: boolean,
```

Comment:
how do we know which of these are non-optional fields? (just wondering; I don't see it in the docs)